ddtrace/appsec/_iast/_evidence_redaction/sql_sensitive_analyzer.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+from ddtrace.appsec._iast.constants import DBAPI_MARIADB
+from ddtrace.appsec._iast.constants import DBAPI_MYSQL
+from ddtrace.appsec._iast.constants import DBAPI_PSYCOPG
+from ddtrace.appsec._iast.constants import DBAPI_SQLITE
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+STRING_LITERAL = r"'(?:''|[^'])*'"
+POSTGRESQL_ESCAPED_LITERAL = r"\$([^$]*)\$.*?\$\1\$"
+MYSQL_STRING_LITERAL = r'"(?:\\\\"|[^"])*"|\'(?:\\\\\'|[^\'])*\''
+LINE_COMMENT = r"--.*$"
+BLOCK_COMMENT = r"/\*[\s\S]*?\*/"
+EXPONENT = r"(?:E[-+]?\\d+[fd]?)?"
+INTEGER_NUMBER = r"(?<!\w)\d+"
+DECIMAL_NUMBER = r"\d*\.\d+"
+HEX_NUMBER = r"x'[0-9a-f]+'|0x[0-9a-f]+"
+BIN_NUMBER = r"b'[0-9a-f]+'|0b[0-9a-f]+"
+NUMERIC_LITERAL = (
+    r"[-+]?(?:" + "|".join([HEX_NUMBER, BIN_NUMBER, DECIMAL_NUMBER + EXPONENT, INTEGER_NUMBER + EXPONENT]) + r")"
+)
+# ORACLE_ESCAPED_LITERAL = r"q'<.*?>'|q'\(.*?\)'|q'\{.*?\}'|q'\[.*?\]'|q'(?\<ESCAPE>.).*?\\k<ESCAPE>'"
+ORACLE_ESCAPED_LITERAL = r"q'<.*?>'|q'\(.*?\)'|q'\{.*?\}'|q'\[.*?\]'|q'([^']*)'"
+
+patterns = {
+    "ANSI": re.compile(
```

Comment:
any reason to not have constants for `ANSI` and `ORACLE` as well?

consistency could be nice, but if they aren't needed anywhere else, I guess 🤷🏻‍♂️ ?